tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -51,8 +49,8 @@ const openTargetDetails = async (targetID) => {
  * @param {string} target.counter
  */
 const expectTargetDetails = async (target) => {
-  expect(await element(by.css('.target-detail h2')).getText()).toEqual(target.title);
-  expect(await element(by.css('.target-detail .cell p')).getText()).toEqual(target.counter);
+  expect(await $('.target-detail h2').getText()).to.equal(target.title);
```

Comment:
Same as previous comments about selectors definitions. Can you do this for all selectors in this document, please?